components/News/TagsSlider.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <v-layout
+    align-center
+    justify-space-between
+    class="tags-container"
+  >
+    <v-sheet
+      class="mx-5"
+      max-width="700"
+      style="background-color: #f2f2f2;"
+    >
+      <v-slide-group>
+        <v-slide-item
+          v-for="tag in selected_tags"
+          :key="tag"
+        >
+          <v-chip
+            class="mx-3 tag-item"
+            outlined
+            close
+            color="primary"
+            @click:close="remove(tag)"
+          >
+            <strong>#{{ tag }}</strong>&nbsp;
+          </v-chip>
+        </v-slide-item>
+      </v-slide-group>
+    </v-sheet>
+    <div>
+      <v-menu
+        offset-y
+        transition="slide-y-transition"
+        bottom
+        :close-on-content-click='false'
+      >
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn
+            class=""
+            outlined
+            rounded
+            color="black"
+            dark
+            v-bind="attrs"
+            v-on="on"
+          >
+            Показать по тегам
+          </v-btn>
+        </template>
+        <v-list class="select-tags">
+          <v-text-field
+            v-model.number="input"
+            placeholder="Введите тэг"
+            type="text"
+            min="0"
+            rounded
+            outlined
+            hide-details
+            color="black"
+            class="sadaka-input"
+          />
+          <v-checkbox
+            v-for="tag in tags"
+            :key="tag"
+            :label="'#'+tag"
+            @click="change(tag)"
+          />
+          <div class="">
+            <v-btn
+              text
+              color="red"
+              @click="remove_all()"
```

Comment:
The parenthesis can be omitted in the template when there are no parameters passed and it is not part of an expression
```suggestion
              @click="remove_all"
```